server/src/test/java/org/apache/cassandra/sidecar/cdc/CDCLogCacheTest.java

```diff
@@ -116,13 +118,18 @@ private File instance1CommitLogFile()
     private CdcLogCache cdcLogCache()
     {
         ExecutorPools executorPools = ExecutorPoolsHelper.createdSharedTestPool(Vertx.vertx());
-        return new CdcLogCache(executorPools, instancesMetadata, 100L);
+        SecondBoundConfiguration mockCacheExpiryConfig = mock(SecondBoundConfiguration.class);
```

Comment:
Can you please document why you're using a `mock` here rather than just instantiating a DurationSpec? Took me a moment to realize you're working around the SecondBoundConfiguration's min unit, if I understand correctly.